src/lib.rs

```diff
@@ -402,12 +402,30 @@ impl Uuid {
     /// Note that not all versions can be generated currently and `None` will be
     /// returned if the specified version cannot be generated.
     ///
+    /// To generate a random UUID (`UuidVersion::Md5`), then the `v3`
```

Comment:
I think we should note in the docs that we're generating these random uuids off the well-known dns namespace uuid.

Reply:
Hmm actually we probably don't need to, since we have enough randomness in the hash we jam into the uuid the base shouldn't matter so much.